schemas/blocks/list_collector.json

```diff
@@ -66,6 +75,9 @@
       },
       "summary": {
         "$ref": "list_collector.json#/summary"
+      },
+      "hide_on_section_summary": {
```

Comment:
This will also be needed on the primary person list collector, and possibly other blocks as there are some questions being added to the who lives here section in census that won't be displayed on the summary.

Reply:
As discussed offline, I've removed this now :)